tests/install_upgrade_operators/console_cli_download/conftest.py

```diff
@@ -17,7 +20,18 @@
 
 
 @pytest.fixture()
-def virtctl_console_cli_downloads_spec_links(admin_client):
+def virtctl_console_cli_downloads_spec_links_scope_function(admin_client):
```

Comment:
this fixture contains one action and used only in one place, do you really need it?